app/javascript/css/table.scss

```diff
@@ -225,3 +225,7 @@ table {
     margin: 0
   }
 }
+
+table caption {
```

Comment:
this is overriding another selector in this stylesheet, should we combine them?

Reply:
good catch, just fixed